cloud/google/gceutil/gceutil.go

```diff
@@ -28,25 +28,40 @@ import (
 	"google.golang.org/api/compute/v1"
 )
 
-// CoreOSImageURL returns the URL of the latest stable CoreOS image for running on Google Compute Engine.
+// CoreOSImageURL returns the URL of the latest stable CoreOS image for running
+// on Google Compute Engine.
 func CoreOSImageURL(cl *http.Client) (string, error) {
-	resp, err := cl.Get("https://www.googleapis.com/compute/v1/projects/coreos-cloud/global/images")
+	return osimageURL(cl, false)
+}
+
+// COSImageURL returns the URL of the latest stable Container-Optimized OS image
+// for running on Google Compute Engine.
+func COSImageURL(cl *http.Client) (string, error) {
+	return osimageURL(cl, true)
+}
+
+func osimageURL(cl *http.Client, cos bool) (string, error) {
```

Comment:
"os" and "image" are separate words, so `osimageURL` should be `osImageURL` per https://golang.org/s/style#mixed-caps.

Reply:
Done